airflow/providers/cncf/kubernetes/pod_generator.py

```diff
@@ -56,6 +56,16 @@
 from airflow.utils.hashlib_wrapper import md5
 from airflow.version import version as airflow_version
 
+try:
+    from airflow.providers.cncf.kubernetes.executors.kubernetes_executor_types import (
+        POD_EXECUTOR_DONE_KEY,
+    )
+except ImportError:
+    # avoid failing import when Airflow pre 2.7 is installed
+    from airflow.kubernetes.kubernetes_executor_types import (  # type: ignore[no-redef]
+        POD_EXECUTOR_DONE_KEY,
+    )
```

Comment:
```suggestion
from airflow.providers.cncf.kubernetes.executors.kubernetes_executor_types import POD_EXECUTOR_DONE_KEY
```

The min version for providers is 2.7 now, so no need for a fallback.